lib/licenses.nix

```diff
@@ -458,4 +458,8 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName = "Zope Public License 2.1";
   };
 
+  dynect = {
+    fullName = "Dyn Inc, Integration Team Deliverable";
```

Comment:
the licenses are ordered alphabetically. Could you move this to the correct position?